rules/framework.bzl

```diff
@@ -279,6 +279,8 @@ def _get_framework_files(ctx, deps):
                     header_in.append(hdr)
                     destination = paths.join(framework_dir, "Headers", hdr.basename)
                     header_out.append(destination)
+                elif hdr.path.endswith(".modulemap") and hdr.owner != dep.label:
```

Comment:
So we're guaranteed that the owner isn't going to be the `dep`? Would you mind dropping a comment about why this is so? If we make a dep of some modulemap rule is that going to work?